soroban-env-host/src/host/error.rs

```diff
@@ -202,21 +202,6 @@ impl<T> TryBorrowOrErr<T> for RefCell<T> {
 }
 
 impl Host {
-    /// Convenience function that only evaluates the auxiliary debug arguments
```

Comment:
If you're going to remove this, remove the doc-comment that references it on the `err!` macro.

Reply:
Removed it in the comment.